sdk/python/tests/integration/permissions/test_auth_permission.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+
+@pytest.mark.integration
+def test_oidc_auth_permissions(start_keycloak_server, temp_dir):
+    print("starting the test")
+    ## TODO:
```

Comment:
well, this is a tough goal, if you want to test an E2E flow b/w remote client(s) and server(s), you'll probably face many issues with running all the components at the same time. also, I don't think there is another existing test like that, even w/o the permissions part. 
the plan misses point 6 abput the offline server and all the tests should run using any of the 3 supported auth types, BTW.

please carefully consider the risks you'll face and the time constraints (don't ask me, but I think there are 😬 ). wouldn't be easier to test only one client-server interaction at a time?